standard_rules/admin_assigned.py

```diff
@@ -0,0 +1,23 @@
+import panther_event_type_helpers as event_type
+
+
+def rule(event):
+    # filter events on unified data model field
+    if event.udm('event_type'):
+        return event.udm('event_type') == event_type.ADMIN_ROLE_ASSIGNED
+    return False
```

Comment:
Would this be a simpler approach:

```
return event.udm('event_type') == event_type.ADMIN_ROLE_ASSIGNED
```

Or am I missing something? It seems that as long as `event_type.ADMIN_ROLE_ASSIGNED` is never `None`  this should be fine.

Reply:
Good point. I'll make this change everywhere relevant.  I think I was initially thinking of saving time by not calling `udm()` but that doesn't make sense at all because it is being called in the `if` statement.  Regardless... making the change!